lib/active_admin/filters/resource_extension.rb

```diff
@@ -174,8 +174,21 @@ def search_status_section
                 else
                   active.filters.each do |filter|
                     li do
-                      span filter.body
-                      b filter.value
+                      if filter.body =~ /ID/i
+                        begin
+                          klass = filter.body.split(' ID').first.gsub(' ', '').constantize
+                          method = klass.inspect.scan(/(title|name|value|id)/).last[0].to_sym
```

Comment:
@Shoroh Just curious whats with the `(title|name|value|id)` regex? Are you searching for a display name to use? If that's the case, we can just assume they're using the default `#display_name` method that ActiveAdmin requires.